Resources/Prototypes/_Impstation/Entities/Mobs/Species/decapoid.yml

```diff
@@ -155,6 +155,7 @@
     needsHands: false # they drag with the claw, or their back legs
     applySpeedModifier: false # they're strong as fuck. also i made this variable just for them.
   - type: Inventory
+    speciesId: gray
```

Comment:
were decapoids supposed to share a species id with grays? thats scary..

Reply:
OOPS this explains a lot. reason why you never make one more change after telling yourself to take a break